### ISSUE TITLE
[Quest] A Dish Best Served Huge - Cape of Stranglethorn

### ISSUE THREAD

**Issue by Repsidian** [WoWManiaUK/Blackwing-Lair#1549]
**Links:**
https://www.wowhead.com/quest=26602/a-dish-best-served-huge
from WoWHead or our Armory

**What is happening:**
cant place the giant feast down to lure the giant out to kill him in order to complete the quest

**What should happen:**
should be able to place the giant food on the floor near the broken boat to lure the giant

**Comment by zippoking** [comment]
Actually you are supposed to click the boat. But you can´t interact with the boat.

How it is:
![A dish best served huge - how it is](https://user-images.githubusercontent.com/18005338/58894242-eb093400-86f1-11e9-9bba-3376934451c4.png)

How it should be:
![A dish best served huge - how it should be](https://user-images.githubusercontent.com/18005338/58894240-eb093400-86f1-11e9-9779-1fc24494c88d.png)

**Comment by Kadajj** [comment]
No interaction with the boat!
![WoWScrnShot_041220_150308](https://user-images.githubusercontent.com/38867875/79070589-c648fa00-7cce-11ea-9893-0f98d4eba4ee.jpg)

once Negolash (ID:1494) spawned, quest works as normal.

![WoWScrnShot_041220_150448](https://user-images.githubusercontent.com/38867875/79070625-08723b80-7ccf-11ea-9f79-133594695b23.jpg)

**Comment by DorizarN** [comment]
But you have to interact with the boat to make him Spawn? I've camped the place for a while now and he's not spawning.

**Comment by Echinaceax** [comment]
yes @DorizarN you have to click the boat first to activate him 
look at 1:15 on the video https://www.wowhead.com/quest=26602/a-dish-best-served-huge#videos:id=51349

**Comment by DorizarN** [comment]
Yes i know i have to click on the boat :P Hence my comment, on this Server you can't click the boat, doesn't let u interact with it.

**Comment by Echinaceax** [comment]
Yes i know you cant click on the boat hence why the report for fix is listed here
have to wait for coders to fix it, then you will be able to click it
